emuu-client/src/components/home/Home.js

```diff
@@ -20,91 +31,102 @@ const theme = createTheme({
 });
 
 function Home() {
+  const [topVideos, setTopVideos] = useState([]);
+  const [recentVideos, setRecentVideos] = useState([]);
+
+  async function getVideos() {
+    //Get all video data
+    const querySnapshot = await getDocs(collection(db, "Videos"));
+
+    //Create array for top videos and sort by likes
+    const querySnapshotTop = [];
```

Comment:
no need to call the array "querySnapshot". Name it something easier to follow through since alot of the variables here are similar.